src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -211,13 +233,14 @@ impl<S: Snapshot> ScannerConfig<S> {
             ts,
             desc,
             bypass_locks: Default::default(),
+            check_has_newer_ts_data: false,
         }
     }
 
     #[inline]
     fn scan_mode(&self) -> ScanMode {
         if self.desc {
-            ScanMode::Backward
+            ScanMode::Mixed
```

Comment:
Why change it?

Reply:
The purpose here is to use near_seek later(we seek twice on one key, the first time is for checking can_be_cached, the second time is for getting the result, we want to use near_seek on the second time), If scan_mode is Backward, we can't call near_seek